### PR TITLE
Newsletter-295: translate into Chinese

### PR DESCRIPTION
**Pull request by hulatown**

### CODE REVIEW

File: _posts/zh/newsletters/2024-03-27-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+- [<!--how-does-the-initialblockdownload-field-in-the-getblockchaininfo-rpc-work-->initialblockdownload 字段在 getblockchaininfo RPC 中是如何工作的？]({{bse}}122169) Pieter Wuille 指出，在节点启动之后，`initialblockdownload` 变成假值要满足两个条件：
+
+  1. “当前使用的链拥有至少跟软件硬编码的常量一样高的累积工作量证明”
+  2. “当前使用的链的最新区块的时间戳在本地时间的 24 小时以内”
+
+## 新版本和候选版本
+
+*热门比特币基础设施项目的新版本和候选版本。请考虑升级到新版本或帮助测试候选版本。*
+
+- [Bitcoin Core 26.1rc2][] 是这个主流全节点实现的一个维护版本的候选发布。
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2024-03-27-newsletter.md:67]: Formatting: newline should be after this line I think

**Review by bitschmidty — COMMENTED** [review]
Links look ok, one suggestion

File: _posts/zh/newsletters/2024-03-27-newsletter.md (hunk 1 above)
Comment (hulatown) [reply _posts/zh/newsletters/2024-03-27-newsletter.md:67]: done

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @hulatown

**Pull request merged**